File: 백준/Silver/2563. 색종이/색종이.py
n = int(input())
size = n * 100
arr = [[0 for _ in range(101)]for _ in range(101)] 
for i in range(n):
    x,y=list(map(int,input().split()))
    for row in range(x,x+10):
        for col in range(y,y+10):
            arr[row][col] = 1

result = 0

for i in arr:
    result+=i.count(1)
print(result)

File: 백준/Silver/2563. 색종이/색종이.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();
        int[][] bg = new int[100][100];

        for (int i = 0; i < n; i++) {
            String temp = sc.nextLine();
            int[] arr = Arrays.stream(temp.split(" ")).mapToInt(Integer::parseInt).toArray();
            for(int j = arr[0]; j < arr[0]+10; j++){
                for(int k = arr[1]; k < arr[1]+10; k++){
                    bg[j][k] = 1;
                }
            }
        }

        int count = 0 ;
        for (int i = 0; i < 100; i++) {
            for (int j = 0; j < 100; j++) {
                if (bg[i][j] == 1) {
                    count++;
                }
            }
        }
        System.out.println(count);
    }
}